source/_includes/installation/operating_system.md

```diff
@@ -112,9 +112,10 @@ sudo apt install libfuse2
 {% if page.installation_type == 'odroid' %}
    If you are using a [Home Assistant Blue](/blue) or ODROID N2+, you can [attach your device directly](/common-tasks/os/#flashing-an-odroid-n2).
 {% endif %}
-2. Download and start <a href="https://www.balena.io/etcher" target="_blank">Balena Etcher</a>. (You may need to run it with administrator privileges on Windows).
+2. Download and start <a href="https://www.balena.io/etcher" target="_blank">Balena Etcher</a>. (You may need to run it with administrator privileges on Windows. For ARM Apple devices it is recommended to use <a href="https://www.raspberrypi.com/software/" target="_blank">Raspberry Pi Imager</a>).
```

Comment:
This is generic documentation, also used for non-Raspberry Pi devices. This thus will now also show up for, for example, the ODROID N2+.